stock_picking_report_delivery_cost/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Delivery cost in Stock Picking Report',
```

Comment:
The module shows cost info in delivery slip and picking operation reports... so the name and summary would be more generic...

Reply:
Well, it's generic right now, but not plural. Let's put `Delivery cost in Picking Reports` (any way, my original idea was to put it only in delivery slip, but end it up putting it on both as sometimes the other is the preferred one).